src/hardware/gameblaster.h

```diff
@@ -36,7 +36,7 @@
 
 #include "mame/emu.h"
 #include "mame/saa1099.h"
-#include "../libs/residfp/resample/TwoPassSincResampler.h"
+#include <residfp/resample/TwoPassSincResampler.h>
```

Comment:
I'm unsure about the value and necessity of this change. Can you explain why this is an improvement in your view?

This blurs the distinction between system-provided and bundled libraries; I actually _prefer_ to make that distinction explicit.

Sure, if you move files around, you need to readjust the relative paths. But how often do we do that? Almost never, and even then it's a 2 min job to adjust the paths.

Reply:
Good points, I agree that not having to use relative paths is nicer. My problem was mainly about the angled bracket vs double-quote convention. Ultimately, I want to keep this convention:

- Angle brackets are used for standard C and C++ libraries and 3rd party dependencies not embedded into the project (e.g. `#include <SDL.h>`) 
- Double-quotes are used for everything else, basically files that live inside the repo. That includes 3rd party dependencies embedded into the repository.

If switching to double-quotes would still work, I'm all for this!